ctapipe/io/simteleventsource.py

```diff
@@ -819,9 +823,16 @@ def _generate_events(self):
                     self.calib_scale,
                     self.calib_shift,
                 )
+
+                pixel_status = self._get_r1_pixel_status(
+                    tel_id=tel_id,
+                    selected_gain_channel=selected_gain_channel,
+                )
                 data.r1.tel[tel_id] = R1CameraContainer(
+                    event_type=trigger.event_type,
```

Comment:
How can you add extra inputs to the `R1CameraContainer` without updating the definition of the container?

Reply:
It is updated? Lines 570ff (new line numbers)

Reply:
Oh, yeah! sorry. It was folded by github and when I tried to check the definition it took me to the old version.